0-bootstrap/README.md

```diff
@@ -22,6 +22,25 @@ Further details of permissions required and resources created, can be found in t
 1. Copy the backend by running `cp backend.tf.example backend.tf` and update `backend.tf` with your bucket from the apply step (The value from `terraform output gcs_bucket_tfstate`)
 1. Re-run `terraform init` agree to copy state to gcs when prompted
     1. (Optional) Run `terraform apply` to verify state is configured correctly
+
+### If you are using Jenkins
+
+If you are using the `jenkins_bootstrap` sub-module, please see the [README](./modules/jenkins-agent/README.md) for the requirements and instructions on how to run the bootstrap step, which include implementing VPN, configuring your Jenkins Master among other steps.
+
+### Update all example backends with your bucket information
```

Comment:
This is out of date with latest code. There should only be one file called `backend.tf.example`, the rest are all `backend.tf` now. 

See latest docs on master https://raw.githubusercontent.com/terraform-google-modules/terraform-example-foundation/develop/0-bootstrap/README.md 

These steps are now optional, for when you are running terraform locally.